server/mailer.js

```diff
@@ -0,0 +1,9 @@
+const sgMail = require('@sendgrid/mail');
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const sendContact = (email, title, message) => {
+  const msg = { ... }
+  sgMail.send(msg)
+}
+
+module.export = { sendContact }
```

Comment:
Nice. Now you have `sendContact` function available for other Node modules.
You could create _server/mailer.runner.js_ file with the following content

~~~js
const { sendContact } = require('./mailer.js')

const email = 'sbbayramof@gmail.com';

sendContact(email, 'TEST', 'TEST');
~~~

You should be able to run it with 

~~~sh
SENDGRID_API_KEY=<your key> node ./server/mailer.runner.js
~~~

Reply:
Gives an error message:

bash: syntax error near unexpected token `newline'